examples/test/src/test/java/org/parceler/ParcelTest.java

```diff
@@ -130,6 +132,13 @@ public void testIntArrayFactoryOrder() {
         assertArrayEquals(new int[]{1, 2, 3, 4, 5}, unwrapped.getArr());
     }
 
+    @Test
+    public void testParcelableClassWithoutAnnotation() {
```

Comment:
Yeah, that's a fine test for this.  `Intent` seems like a strange use case to test... why not test a specific inner class Parcelable?